### ISSUE TITLE
Scale font size

### ISSUE THREAD

**Issue by bhollis** [DestinyItemManager/DIM#9862]
DIM's default font size is quite small, which is great for how much info we try to pack in, but can be hard to see on certain screen densities. We should:

1. Use `font: -apple-system-body;` to set the base font size based on Apple's dynamic type basis.
2. Set font sizes using relative units so they scale based on an initial size, probably with some minimums. https://www.npmjs.com/package/postcss-pixels-to-rem can help with this.
3. Offer a setting for font size adjustment independent of the system font size.

cc @guise @vivekhnz

**Comment by guise** [comment]
Couple of thoughts

- Setting a root `[font-size: 62.5%;`](https://www.aleksandrhovhannisyan.com/blog/62-5-percent-font-size-trick/) should allow easy conversion of px → rem (12px → 1.2rem) 
    - This might negate the need for a post-processor?
- Not sure if icon fonts should be exempt by default or handled case-by-case
- I suspect particularly space-constrained components might need text-overflow handling added for larger fonts (e.g. power level on polaroids), but this could end up clipping even more content with an `…` 
    - Alternatively it might be better to adjust layouts in these situations (e.g. stack the polaroid icons above the power level text)
